airflow/providers/airbyte/CHANGELOG.rst

```diff
@@ -19,13 +19,27 @@
 Changelog
 ---------
 
+2.1.1
+.....
+
+Misc
+~~~~
+
+* ``Optimise connection importing for Airflow 2.2.0``
+
+.. Below changes are excluded from the changelog. Move them to
+   appropriate section above if needed. Do not delete the lines(!):
+   * ``Fix messed-up changelog in 3 providers (#17380)``
+   * ``Update description about the new ''connection-types'' provider meta-data (#17767)``
+   * ``Import Hooks lazily individually in providers manager (#17682)``
+
 2.1.0
 .....
 
 Bug Fixes
 ~~~~~~~~~
 
-* ``Fixed wrongly escaped characters in amazon&#39;s changelog (#17020)``
+* ``Fixed wrongly escaped characters in amazon's changelog (#17020)``
```

Comment:
mmm this description is kinda weird for the airbyte provider.
https://github.com/apache/airflow/pull/17020 changed files in several providers. 
I think It would be better change all occurrences of:
`Fixed wrongly escaped characters in amazon's changelog (#17020)`
with
`Fixed wrongly escaped characters in changelog (#17020)`

Reply:
Actually .... it should not be there at all :). It's the kind of change that should go to "ignored" part. below. I will move it